server/controllers/conversation.js

```diff
@@ -0,0 +1,73 @@
+const asyncHandler = require("express-async-handler");
+const Conversation = require("../models/Conversation");
+const Profile = require("../models/Profile");
+
+// @route GET /conversations
+// @desc get all conversations for the authenticated user
+// @access private
+exports.getAllConversations = asyncHandler(async (req, res, next) => {
+  const userId = req.user.id;
+  const profile = await Profile.findOne({ user: userId });
+
+  const conversations = await Conversation.find({
+    members: { $in: [profile._id] },
+  }).populate("members", "firstName lastName");
+
+  res.status(200).json({ conversations });
```

Comment:
It would been nice to have the last message data sent along with a conversation :)

Reply:
it's applied in the next [PR ](https://github.com/hatchways/team-dumplings/pull/73)
thank you